telemetry/roles/telemetry_validation/vars/main.yml

```diff
@@ -155,3 +155,14 @@ provision_credentials_vault_path: "{{ role_path }}/../../../input/.provision_cre
 ansible_vault_search_key: "$ANSIBLE_VAULT;"
 provision_config_credentials_syntax_fail_msg: "Failed. Syntax errors present in provision_config_credentials.yml. Fix errors and re-run playbook again."
 conf_file_mode: "0644"
+
+# Usage: validate_idrac_inventory.yml
+bmc_group_data_filename: "/opt/omnia/telemetry/bmc_group_data.csv"
+bmc_group_data_file_not_found_msg: "Failed. The file BMC group data file does not exist."
```

Comment:
Add file path in error message. add message to run discovery_provision if file not found